jina/peapods/peas/jinad.py

```diff
@@ -215,15 +212,55 @@ def __init__(self, args: 'argparse.Namespace'):
             target=JinaDProcessTarget(),
             kwargs={
                 'args': args,
-                'name': self.name,
```

Comment:
why is `envs` removed?

Reply:
Ah of course, this should be handled in the `Pea` in the other end of `JinaD` I guess

Reply:
This was a miss. It should be passed from `JinaDPea` to `JinaDProcessTarget` to `remote Pea`